README.md

```diff
@@ -1,69 +1,46 @@
 # S4S Testsuite
 
-Framework for testing S4S API implementations
+Framework for testing S4S API implementations. The interface is a [Flask](http://flask.pocoo.org/) application, and the tests themselves are run as [Celery](http://www.celeryproject.org/) tasks. The tests are written using the [behave](http://behave.readthedocs.io/en/stable/index.html) framework.
 
-## Build and run in docker
+## S4S reference stack
+
+The test suite is one component of the [Sync for Science reference stack](https://github.com/sync-for-science/reference-stack-docker), which contains additional services such as DSTU2 and STU3 FHIR servers. The `SMART EHR` servers in the vendor list of the test suite refer to the servers that run with the reference stack, so they will be unavailable if you choose to run the test suite independently. *Additionally*, the FHIR resource validation steps will fail when the test suite is run independently, because those FHIR servers are used to perform the validation.
```

Comment:
Say "independently (i.e., outside of the full reference stack)" the first time, just because a possible mis-reading of "independently" is "on your own, without our help".